scripts/postgres_test_setup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Create the Docker container
+docker run --name aqueduct-postgres -e POSTGRES_PASSWORD=aqueduct -d postgres -p 5432:5432
```

Comment:
The Docker Postgres image (https://hub.docker.com/_/postgres) has the following environment variable:
```
POSTGRES_DB
This optional environment variable can be used to define a different name for the default database that is created when the image is first started. If it is not specified, then the value of POSTGRES_USER will be used.
```
You might be able to just run `docker run --name aqueduct-postgres -e POSTGRES_PASSWORD=aqueduct -e POSTGRES_DB=aqueducttest -d postgres -p 5432:5432`

Reply:
@eunice-chan Great catch here, I've updated the setup script to just be one line :)